packages/easy-client/src/adapter.ts

```diff
@@ -19,7 +19,7 @@ export function adapter (signer: Signer, api: Api): (config: RequestConfig) => A
 
     // sign and async response
     // method
-    config.method = (config.method && config.method.toUpperCase()) || 'GET'
+    config.method = config.method || 'GET'
```

Comment:
`toUpperCase` is unnecessary.
https://github.com/axios/axios/blob/v0.19.0/lib/core/Axios.js#L38